tests/sanity_helpers.py

```diff
@@ -0,0 +1,128 @@
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants, node
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.cluster import CephCluster
+
+from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+def health_check(nodes):
+    """
+    Perform Ceph and cluster health checks
+    """
+    assert node.wait_for_nodes_status(nodes), (
+        f"Not all nodes reached status {constants.NODE_READY}"
+    )
+    ceph_cluster = CephCluster()
+    assert ceph_health_check(
+        namespace=config.ENV_DATA['cluster_namespace']
+    )
+    ceph_cluster.cluster_health_check(timeout=60)
```

Comment:
According to the documentation, `cluster_health_check` can return False in case the cluster isn't healthy. I didn't manage to find where/how, but I think it can be a good idea to assert this line regardless.
```suggestion
    assert ceph_cluster.cluster_health_check(timeout=60)
```

Reply:
it will raise exceptions.CephHealthException. see https://github.com/red-hat-storage/ocs-ci/blob/1bdbb94753bb4742d534658c64f1d433a83c054e/ocs_ci/ocs/cluster.py#L198

https://github.com/red-hat-storage/ocs-ci/blob/1bdbb94753bb4742d534658c64f1d433a83c054e/ocs_ci/ocs/cluster.py#L212

https://github.com/red-hat-storage/ocs-ci/blob/1bdbb94753bb4742d534658c64f1d433a83c054e/ocs_ci/ocs/cluster.py#L219